x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -1088,8 +1094,12 @@ private LFUCacheEntry initChunk(LFUCacheEntry entry) {
                 assignToSlot(entry, freeSlot);
             } else {
                 // need to evict something
+                int frequency;
                 synchronized (SharedBlobCacheService.this) {
-                    maybeEvict();
+                    frequency = maybeEvict();
+                }
+                if (frequency > 0) {
+                    blobCacheMetrics.getEvictedCountNonZeroFrequency().incrementBy(frequency);
```

Comment:
`.incrementBy(frequency)`: Is this correct?  We want the "count of entries that were evicted with non-zero frequency", not "the frequency of things that were evicted", right?

Reply:
Good catch!